setup.py

```diff
@@ -42,6 +41,18 @@
 
 
 EXTRAS_REQUIRE = {
+    "onnxruntime": "onnxruntime>=1.11.0",
+    "onnxruntime-gpu": "onnxruntime-gpu>=1.11.0",
+    "onnxruntime-training": [
+        "evaluate",
+        "torch-ort",
+        "accelerate",
+        "datasets>=1.2.1",
+        "protobuf>=3.20.1",
+        "onnxruntime-training>=1.11.0",
+    ],
+    "exporters": "onnxruntime>=1.11.0",
+    "exporters-gpu": "onnxruntime-gpu>=1.11.0",
```

Comment:
thinking about removing the two as they are redundant with onnxruntime/onnxruntime-gpu https://github.com/huggingface/optimum/blob/main/setup.py#L72